rest_golang/db/data.go

```diff
@@ -115,3 +120,68 @@ func GetActivityStats(client *mongo.Client, filter model.StatsFilter) ([]model.A
 
 	return activityStats, nil
 }
+
+func InsertActivityStats(client *mongo.Client, user model.User) error {
+	var err error
+	//Create a handle to the respective collection in the database.
+	collection := client.Database("wondervilleDev").Collection("activityStats")
+	//Check to see if activity is in the DB, if so update hit counter else
+	//Perform InsertOne operation & validate against the error.
+	incomingActivityName := user.Payload.Asset.Name
+
+	filter := bson.M{"name": incomingActivityName}
+
+	cursor, err := collection.Find(context.TODO(), filter)
+
+	if err != nil {
+		panic(err)
+	}
+
+	var res []bson.M
+
+	if err = cursor.All(context.TODO(), &res); err != nil {
+		panic(err)
+	}
+
+	if len(res) > 0 {
+		fmt.Println("Finding hit for exisiting activity")
```

Comment:
we probably only need to keep the error logs

Reply:
Addressed